literature.bib

```diff
@@ -4480,6 +4601,78 @@ @article{TKB+:SCP14
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2013 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{SLSA:SIGPLAN13,
+	author = {Schulze, Sandro and Liebig, J\"{o}rg and Siegmund, Janet and Apel, Sven},
+	title = {{Does the Discipline of Preprocessor Annotations Matter?: A Controlled Experiment}},
+	journal = SIGPLAN,
+	volume = {49},
+	number = {3},
+	year = 2013,
+	issn = {0362-1340},
+	pages = {65--74},
+	numpages = {10},
+	url = {http://doi.acm.org/10.1145/2637365.2517215},
+	doi = {10.1145/2637365.2517215},
+	acmid = {2517215},
+	publisher = ACM,
+	address = NY,
+	keywords = {c preprocessor, controlled experiment, disciplined annotations, program comprehension, variability},
+}
+
+@inproceedings{KKT:ACM13,
+	author = {T. {Kehrer} and U. {Kelter} and G. {Taentzer}},
+	booktitle = ASE,
+	publisher = ACM,
+	title = {{Consistency-Preserving Edit Scripts in Model Versioning}},
+	year = 2013,
+	volume = {},
+	number = {},
+	pages = {191--201},
+	tt-tags = {semantic lifting},
+	keywords = {configuration management, software tools, consistency preserving edit scripts, model versioning, software development, internal model representations, automatic procedure, model merging, model comparison, model patching, software tools, Unified modeling language, Merging;Semantics, Concrete, Adaptation models, Abstracts, Syntactics},
+	doi = {10.1109/ASE.2013.6693079}
+}
+
+@article{RBS:IST13,
+	title = {{Software Clone Detection: A Systematic Review}},
+	author = {Rattan, Dhavleesh and Bhatia, Rajesh and Singh, Maninder},
+	journal = IST,
+	volume = {55},
+	number = {7},
+	pages = {1165--1199},
+	year = 2013,
+	publisher = Elsevier
+}
```

Comment:
Again, where do you find these? It takes me ten minutes per reference to **not** find it. Just as IEEE, Elsevier does not provide his information, too. 👎 :(

Reply:
Welcome to research. This is why I at least want to avoid that as duplicated effort.

The problem is that some publication templates actually require this and it would be good to have this consistent too. Try to find it on ACM or DBLP or the publisher. If that does not help either, just document your effort by adding a comment. I've done this before.

Reply:
Ok thanks for the advise.